proto/noble/dollar/portal/v1/events.proto

```diff
@@ -20,6 +20,13 @@ message PeerUpdated {
   bytes new_manager = 5;
 }
 
+// SupportedBridgingPathSet is an event emitted whenever a supported bridging path is set.
+message SupportedBridgingPathSet {
+  uint32 destination_chain_id = 1 [(gogoproto.casttype) = "uint16"];
```

Comment:
I think this is the one thing I keep coming back to in this PR. Why do we need to store something as a `uint32` if we end up using it as a `uint16` all over the place? Wouldn't it be easier to use `uint16` in the definition as well?

Reply:
See https://github.com/noble-assets/dollar/pull/21#discussion_r1963300497 😄